helm-charts/common/vllm-openvino/README.md

```diff
@@ -0,0 +1,68 @@
+# OpenVINO vLLM 
+
+Helm chart for deploying OpenVINO optimized vLLM Inference service.
```

Comment:
As mentioned earlier, please try using the vllm helm charts with vllm-openvino.yaml values.

Reply:
added a separate values file for OpenVINO.